app/controllers/concerns/foreman/controller/registration.rb

```diff
@@ -109,6 +110,10 @@ def context_urls
     { registration_url: registration_url }
   end
 
+  def registration_host
+    registration_url.host
```

Comment:
Isn't this also available within the template?

Reply:
It is but there was some safe mode related errors I was running into accessing this directly in the template..

Reply:
Ah yes, we don't allow the URI object in safemode.